serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc

```diff
@@ -0,0 +1,20 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-accessing-secrets-configmaps"]
+= Accessing secrets and ConfigMaps from Serverless functions
+include::modules/common-attributes.adoc[]
+:context: serverless-functions-secrets
+
+toc::[]
+
+From your functions, you can access data stored in `Secret` objects or in ConfigMaps. Such data can be either mounted as volumes or assigned to environment variables. You can configure this manually or with an interactive program.
+
+[NOTE]
+====
+Accessing data stored in secrets or ConfigMaps can only be used when the function is deployed in the cluster. It does not work with a function running locally.
```

Comment:
config maps

I'd make this note important or else write it into the intro, e.g.

"You can access data that is stored in secrets and config maps for functions that are deployed to your cluster. This data can be mounted as volumes, or assigned to environment variables. You can mount volumes or assign environment variables by using the Knative CLI `kn func` commands. These options are not available for functions which are running locally."

I'm not sure I'd mention the manual vs interactive thing, I'd just cover the prompt options as part of the procedure as per the example above for creating functions

Reply:
Changed the introduction inspired by your suggestion.

"I'm not sure I'd mention the manual vs interactive thing" it sets stage for the following sections (the first one is about manual configuration), so I'd rather keep the sentence about manual vs using kn CLI.